application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/MissingLicensesUIX.xml

```diff
@@ -120,6 +223,22 @@
     !$xwiki.exists('XWiki.Alerts.Code.AlertsMenuUIX'))
   {{include reference="Licenses.Code.LicensesNotificationsUIX" /}}
 #end
+
+#set ($unlicensedSheets = $services.licensing.licensor.getUnlicensedSheets($doc.documentReference, $xcontext.action))
+#if ($services.sheet.getSheets($doc, $xcontext.action).isEmpty() &amp;&amp; $unlicensedSheets.size() &gt; 0)
+  #set ($extensions = $collectionstool.set)
+  #foreach ($sheet in $unlicensedSheets)
+    #foreach ($extension in $services.extension.xar.getInstalledExtensions($sheet))
+      #set ($discard = $extensions.add($extension.name))
+    #end
+  #end
+  {{html clean="false"}}
+  &lt;div class="hidden" id="licenseCheckingExtensionPoint"&gt;
```

Comment:
I would add a comment explaining why we hide the message and then move it with JavaScript (since it's not obvious and it's easy to forget in 1 month): because the "After content header" UIX (``org.xwiki.platform.template.content.header.after``) is available only since XWiki 10.4RC1 and we need to support XWiki 8.4+